src/SearchResults.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import moment from "moment";
+
+const headings = [
+  `id`,
+  `title`,
+  `firstName`,
+  `surname`,
+  `email`,
+  `roomId`,
+  `checkInDate`,
+  `checkOutDate`,
+  `numberOfNights`
+];
+
+function SearchResults(props) {
+  return (
+    <table className="table">
+      <thead className="table-header">
+        <tr>
+          {headings.map(heading => {
+            return <th scope="col">{heading}</th>;
+          })}
+        </tr>
+      </thead>
+      <tbody>
+        {props.bookings.map(booking => {
+          return (
+            <tr>
```

Comment:
do you see any warnings in the console about missing keys..? If so, can you fix them using the index as the key?